src/backend/api/src/user_profile.rs

```diff
@@ -0,0 +1,31 @@
+use candid::{CandidType, Deserialize};
+
+#[derive(Debug, Clone, CandidType, Deserialize, PartialEq, Eq)]
+pub enum UserConfig {
+    #[serde(rename = "admin")]
+    Admin { bio: String },
+
+    #[serde(rename = "reviewer")]
+    Reviewer {
+        bio: String,
+        neuron_id: u64,
+        wallet_address: String,
+    },
+
+    #[serde(rename = "anonymous")]
+    Anonymous,
+}
```

Comment:
Why not having a common struct for users' configs? I expect we'll have to share many types across admins and reviewers (like the `bio` now)

Reply:
So do you suggest to use a struct like this maybe?

```
struct UserConfig {
  bio: Option<String>,
  neuron_id: Option<String>,
  wallet_address: Option<String>
}
```

Or do you mean something else?

Reply:
Yeah but now that I see it, it probably doesn't make sense. My only concern was that `Admin` and `Reviewer` will likely share a lot of fields... Let's use the current structure for now.